src/Swarm/Util.hs

```diff
@@ -373,11 +373,14 @@ number _ = plural
 
 -- | Surround some text in single quotes.
 squote :: Text -> Text
-squote t = T.concat ["'", t, "'"]
+squote t = T.concat ["'", escape t, "'"]
 
 -- | Surround some text in double quotes.
 quote :: Text -> Text
-quote t = T.concat ["\"", t, "\""]
+quote t = T.concat ["\"", escape t, "\""]
+
+escape :: Text -> Text
+escape = T.replace "\"" "\\\"" . T.replace "\\" "\\\\"
```

Comment:
I'm not quite sure I understand the motivation for adding `escape` here.  `quote` and `squote` are only used for formatting messages, not for pretty-printing, right?  So why does it matter that we escape quote marks and backslashes?  In fact, I think I'd rather *not* have escape sequences showing up in messages shown to the player.